gis/osmium-tool/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        osmcode osmium-tool 1.13.1 v
+github.tarball_from archive
+
+categories          gis
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+license             GPL-3+ MIT BSD Boost-1
+
+description         A command line tool for working with OpenStreetMap
+
+long_description    A multipurpose command line tool for working with \
+                    OpenStreetMap data based on the Osmium library
+
+checksums           rmd160  3b024c02f3a1ae1a31ab3ef7369579e3d13fe59e \
+                    sha256  d6273e2614d390d8444b767018b7023bdac3538cbe094d2799eee50b6f08cd03 \
+                    size    394224
+
+compiler.cxx_standard 2011
+
+depends_lib         port:libosmium \
```

Comment:
similar to what I said in the other PR: please use `depends_lib-append`

Reply:
Ok